src/commonTest/kotlin/fr/acinq/lightning/blockchain/electrum/WalletStateUnitTest.kt

```diff
@@ -0,0 +1,53 @@
+package fr.acinq.lightning.blockchain.electrum
+
+import fr.acinq.bitcoin.*
+import fr.acinq.lightning.Lightning.randomKey
+import fr.acinq.lightning.tests.utils.LightningTestSuite
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class WalletStateUnitTest : LightningTestSuite() {
+
+    @Test
+    fun `sign a p2wpkh tx`() {
+
+        // this tx pays 0.038 BTC to p2wpkh tb1qtkl49wxh4a8mt7dhtwqg7z5zs3yn2vdn6qcz7p
```

Comment:
Instead of this comment, it would be better to just verify it:

```kotlin
assertEquals(parentTx.txOut.first(), TxOut(3_800_000.sat, Script.pay2wpkh(pub1)))
```

Reply:
Done in [6fcb944](https://github.com/ACINQ/lightning-kmp/pull/350/commits/6fcb944de8b901d3efbb7193f2dd8ec7c1cfe9a5).